src/domain/page/Page.tsx

```diff
@@ -22,12 +23,30 @@ const Page: React.SFC = ({ children }) => {
               key="harbors"
               onClick={() => history.push('/harbors')}
               label={
-                <InternalNavLink
-                  to="harbors"
-                  icon={<Icon name="fence" color="standard" />}
-                >
-                  <Text bold>Venepaikat</Text>
-                </InternalNavLink>
+                <InternalLink to="harbors">
+                  <Button
+                    variant="text"
+                    icon={<Icon name="fence" color="standard" />}
+                    onClick={() => {}}
+                  >
+                    <Text bold>Venepaikat</Text>
+                  </Button>
+                </InternalLink>
+              }
+            ></Expandable>,
+            <Expandable
+              key="customers"
+              onClick={() => history.push('/customers')}
+              label={
+                <InternalLink to="customers">
+                  <Button
+                    variant="text"
+                    icon={<Icon name="individual" color="standard" />}
+                    onClick={() => {}}
+                  >
+                    <Text bold>Asiakkaat</Text>
```

Comment:
Can you move the text here to the `fi.json` file?

Reply:
Yeah, I also added some other missing translations.